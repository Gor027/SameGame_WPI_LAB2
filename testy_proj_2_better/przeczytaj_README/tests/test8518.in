bm bm b m bm mmmbbbb  bbb  m  mm  bm m   bm b mmbb  mmmb m  m  mbm mmb m   
  b mbbm
   mbbmmmm m mbm b bbmbb mbm b
 b  mb mb mbb  b bmb bbmb b bm mm mbm bbbbm mmm bbmmmbmb mbbbm
  
  mmm bbbbb m bm bb mbmm bmb b  b  bbm    b bb 
mmm bmmb m   mb  mbbmmm  b
m mbmmb    mb 
m mb mmmmb bb   m   bmmbmmmmb  b m mmmmbbm bbbb b mmmbm bbm bmmmmmmmm  b  mbbmm
mbbb  bbmmmbb
