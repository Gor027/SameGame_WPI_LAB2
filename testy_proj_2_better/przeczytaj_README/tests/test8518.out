      b
      bm   m          b
      mm   mb    b   mbb
   bm bmmm bm mm bbbbbbb mbm       m        b
   bmbmbmbbmbmbm bmbmbmmbbbm b m   bmm   bm mmm b   mmm  m  m
 bbbbbbbbmmbbmbbbbbbmmmmbbmbbm mm bbbmbbbbm bbm bbmmmbmb mb bm mbm mm m
bbbbbbbbbbbbbmbmmmbbbmmbmmmmbbmbbmmmmmmbbmbbbbbmbbmmmbmbbbmbbmmmmmmmmbbmbbmm
